ztp/extra-manifests-builder/99-crio-disable-wipe/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+MCPROLE=${MCPROLE:-master}
+
+crio_disable_wipe_base64=$(base64 -w0 crio-disable-wipe.toml)
+
+echo "\
```

Comment:
This works - You could cut down on a lot of the boilerplate if you use `mcmaker` like here:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/extra-manifests-builder/04-accelerated-container-startup/build.sh#L8-L9